src/app/index.tsx

```diff
@@ -39,9 +41,14 @@ export default (options: Options = {}) => {
     navigation: navigation.createReducer(history.location),
   });
 
+  const services = {
```

Comment:
so far these are always the same and are created here, in the future we may want to add some to the options passed into this function which creates a dependency injection pattern to be used between browser and pre-renderer